### PR TITLE
Update for controller & charts to publish into latest ecr-public repository

### PR DESCRIPTION
**Pull request by vijtrip2**
Issue: https://github.com/aws-controllers-k8s/community/issues/829

* remove redundant AWS_SERVICE_DOCKER_IMG_TAG variable
* Use VERSION as image tag for controller and chart
* Update the repositories for controller and chart to use new format and value. i.e. "public.ecr.aws/u2r4f3v7/<AWS_SERVICE>-controller:<VERSION>" and "public.ecr.aws/u2r4f3v7/<AWS_SERVICE>-chart:<VERSION>"

### CODE REVIEW

File: cd/scripts/release-controller.sh [hunk 1]
```diff
@@ -89,12 +89,11 @@ if ! (echo "$VERSION" | grep -Eq "stable$"); then
   BUILD_DATE=$(date +%Y-%m-%dT%H:%M)
   CONTROLLER_IMAGE_DOCKERFILE_PATH=$CD_DIR/controller/Dockerfile
 
-  DEFAULT_DOCKER_REPOSITORY="public.ecr.aws/aws-controllers-k8s/controller"
+  #TODO: vijat@ to replace u2r4f3v7 with aws-controllers-k8s
+  DEFAULT_DOCKER_REPOSITORY="public.ecr.aws/u2r4f3v7/$AWS_SERVICE-controller"
```
Comment (RedbackThomson) [inline cd/scripts/release-controller.sh:93]: For every change in this file we'll have to redeploy a new `cd` Prow image. May I suggest that we hardcode the correct public image repository into this variable now - and we can then override more easily by modifying the environment variables in the Prow job?

File: cd/scripts/release-controller.sh [hunk 2]
```diff
@@ -132,8 +131,8 @@ fi
 
 cd "$WORKSPACE_DIR"
 
-DEFAULT_HELM_REGISTRY="public.ecr.aws/aws-controllers-k8s"
-DEFAULT_HELM_REPO="chart"
+DEFAULT_HELM_REGISTRY="public.ecr.aws/u2r4f3v7"
```
Comment (RedbackThomson) [inline cd/scripts/release-controller.sh:134]: Likewise with above comment

File: cd/scripts/release-controller.sh (hunk 1 above)
Comment (vijtrip2) [reply cd/scripts/release-controller.sh:93]: Sounds good.

File: prow/jobs/jinja/postsubmits/controller_release.jinja2 [hunk 3]
```diff
@@ -10,7 +10,12 @@
     spec:
       serviceAccountName: post-submit-service-account
       containers:
-        - image: public.ecr.aws/aws-controllers-k8s/prow:prow-deploy-0.0.2
+        - image: public.ecr.aws/aws-controllers-k8s/prow:prow-deploy-0.0.3
+          env:
+          - name: DOCKER_REPOSITORY
+            value: "public.ecr.aws/u2r4f3v7/{{ service }}-controller"
+          - name: HELM_REGISTRY
+            value: "public.ecr.aws/u2r4f3v7"
```
Comment (jaypipes) [inline prow/jobs/jinja/postsubmits/controller_release.jinja2:18]: :+1:

**Review by jaypipes — APPROVED** [review]
++

**Pull request merged**